man/useradd.8.xml

```diff
@@ -464,6 +464,10 @@
 	    <option>-m</option> options if you want a home directory for a
 	    system account to be created.
 	  </para>
+	  <para>
+	    Note that <command>useradd</command> with this option will not
```

Comment:
Little rewording: `Note that this option...`

Reply:
Thank you. Updated.